templates/test/overview.html.ep

```diff
@@ -134,5 +135,92 @@
                 %= include 'test/overview_result_table', type => $type, type_results => $type_result, type_archs => $type_archs->{$type}
             % }
         % }
+||||||| merged common ancestors
```

Comment:
It seems there's a conflict here.

Reply:
that might explain some errors ;) Updated